src/index.html

```diff
@@ -19,6 +19,101 @@ <h1 class="display-2">LACKER</h1>
         class="shadow p-3 mb-5 bg-white rounded"
       />
       <input type="submit" value="Go!" class="btn btn-primary" />
+
+      <div class="row align-items-center">
+        <div class="col">
+          <div class="card">
+            <img
+              src="https://images.unsplash.com/photo-1591814468924-caf88d1232e1?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80"
+              class="card-img-top"
+              alt="Picture of ramen"
+            />
+            <h5 class="card-title">Ramen</h5>
+            <p class="card-text">
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Sapiente, numquam.
+            </p>
+            <a href="#" class="btn btn-primary">Yes, please!</a>
+          </div>
+        </div>
+        <div class="col">
+          <div class="card">
+            <img
+              src="https://images.unsplash.com/photo-1559314809-0d155014e29e?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
+              class="card-img-top"
+              alt="Picture of pad thai"
+            />
+            <h5 class="card-title">Pad Thai</h5>
+            <p class="card-text">
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Sapiente, numquam.
+            </p>
+            <a href="#" class="btn btn-primary">Yes, please!</a>
+          </div>
+        </div>
+        <div class="col">
+          <div class="card">
+            <img
+              src="https://images.unsplash.com/photo-1586128743915-ec7788189715?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
+              class="card-img-top"
+              alt="Picture of pad krapao"
+            />
+            <h5 class="card-title">Pad Krapao</h5>
+            <p class="card-text">
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Sapiente, numquam.
+            </p>
+            <a href="#" class="btn btn-primary">Yes, please!</a>
+          </div>
+        </div>
+        <div class="row align-items-center">
```

Comment:
Put this ROW one Level up. This ROW is in the FIRST ROW, but should not.

should look like: 
<div class="row">
  <div class="col"></div>
  <div class="col"></div>
  <div class="col"></div>
</div>
<div class="row">
  <div class="col"></div>
  <div class="col"></div>
  <div class="col"></div>
</div>

Reply:
Thanks! This little changed actually fixed it :)